doc/source/ray-observability/user-guides/configure-logging.md

```diff
@@ -136,13 +136,58 @@ The output is as follows:
 (task pid=534174) Hello there, I am a task 0.17536720316370757 [repeated 99x across cluster] (Ray deduplicates logs by default. Set RAY_DEDUP_LOGS=0 to disable log deduplication)
 ```
 
-This feature is especially useful when importing libraries such as `tensorflow` or `numpy`, which may emit many verbose warning messages when imported. Configure this feature as follows:
-
-1. Set ``RAY_DEDUP_LOGS=0`` to disable this feature entirely.
-2. Set ``RAY_DEDUP_LOGS_AGG_WINDOW_S=<int>`` to change the agggregation window.
-3. Set ``RAY_DEDUP_LOGS_ALLOW_REGEX=<string>`` to specify log messages to never deduplicate.
-4. Set ``RAY_DEDUP_LOGS_SKIP_REGEX=<string>`` to specify log messages to skip printing.
-
+This feature is useful when importing libraries such as `tensorflow` or `numpy`, which may emit many verbose warning messages when you import them. 
+
+Configure the following environment variables on the driver process before importing Ray to customize log deduplication:
```

Comment:
I think this "before importing Ray" would need a bold up since people easily ignore this fact, and I wonder could there be a way to suggest a warning somehow during runtime to be foolproof? 
```suggestion
Configure the following environment variables on the driver process **before importing Ray** to customize log deduplication:
```